edge/pkg/edgehub/clients/factory.go

```diff
@@ -13,10 +15,14 @@ const (
 	ClientTypeQuic      = "quic"
 )
 
+// ErrorWrongClientType is Wrong Client Type Error
+var ErrorWrongClientType = errors.New("Wrong Client Type")
```

Comment:
Sorry for nit picking, we are following the convension that error message is started with lower-case word while log message is started with uppper-case word. So in this case, the error string should be "wrong Client Type".

Reply:
ok